### PR TITLE
Close #21610: Update Strings for Tab Settings

### LINKED ISSUE #21610: Update Strings for Tab Settings
This change is to update the following:

**Auto-close open tabs**

    Never

    After one month

    After one week

    After one day

**Move old tabs to inactive**

          Tabs you haven’t viewed for two weeks get moved to the inactive section.

For designs, refer to: https://mozilla-hub.atlassian.net/browse/FNXV2-18080

### PR DESCRIPTION
**Pull request by rocketsroger**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Review by MozillaNoah — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -670,11 +670,9 @@
...
     <!-- Inactive tabs -->
     <!-- Category header of a preference that allows a user to enable or disable the inactive tabs feature -->
-    <string name="preferences_inactive_tabs">Inactive tabs on tab tray</string>
-    <!-- Title for inactive tabs preference -->
-    <string name="preferences_inactive_tabs_title">Make tabs inactive after 14 days</string>
-    <!-- Summary of inactive tabs preference -->
-    <string name="preferences_inactive_tabs_summary">To improve speed, tabs will be moved into the inactive tabs section on your tab tray</string>
+    <string name="preferences_inactive_tabs">Move old tabs to inactive</string>
+    <!-- Title of inactive tabs preference -->
+    <string name="preferences_inactive_tabs_title">Tabs you haven\'t viewed for two weeks get moved to the inactive section.</string>
```
Comment (csadilek) [inline app/src/main/res/values/strings.xml:675]: @rocketsroger it's late and I may be missing something, but why the backslash in `haven\'t`?

Comment (csadilek) [reply app/src/main/res/values/strings.xml:675]: (seems like we don't escape anywhere else, but land when ready)

Comment (rocketsroger) [reply app/src/main/res/values/strings.xml:675]: Confirmed that we do need it.  Will not build without the backslash.  I'll land this.

Comment (rocketsroger) [reply app/src/main/res/values/strings.xml]: Fixed.  Thanks

**Pull request merged**